test/jdk/javax/swing/plaf/basic/BasicTextUI/8001470/bug8001470.java

```diff
@@ -48,21 +53,27 @@ public void run() {
                 JPanel container = (JPanel) frame.getContentPane();
                 container.setLayout(new GridLayout(2,1));
 
-                textField1 = new JTextField("\u0e01");
-                textField2 = new JTextField("\u0c01");
+
+                textField1 = new JTextField("\u2588");
+                textField2 = new JTextField("\u2588");
 
                 container.add(textField1);
                 container.add(textField2);
                 frame.setVisible(true);
                 frame.pack();
             }
         });
-        if( textField1.getHeight() < 10 || textField2.getHeight() < 10 )
+        Thread.sleep(1000);
+        SwingUtilities.invokeAndWait(() -> {
+            height1 = textField1.getHeight();
+            height2 = textField2.getHeight();
+        });
+        if( height1 < 10 || height2 < 10 ) {
```

Comment:
```suggestion
        if ( height1 < 10 || height2 < 10 ) {
```

Reply:
It should be
```java
        if (height1 < 10 || height2 < 10) {
```
No spaces after the opening parenthesis and before the closing one.